jenkins/ci.fedora.Dockerfile

```diff
@@ -81,4 +81,9 @@ RUN \
     `#` \
     `# Save space in the container image.` \
     `#` \
-    dnf -y clean all && rm -rf /var/cache/dnf
+    dnf -y clean all && rm -rf /var/cache/dnf && \
+    `#` \
+    `# Tweak npm.` \
```

Comment:
The sequence here looks a bit odd, when the "Tweak npm" comment precedes the installation of npm. "Tweak" is also a bit vague, when presumably it refers to increasing the timeout so we can install all the dependencies.

Reply:
Words are so hard....

From where I sit, `npm` was _installed_ on line 74; line 88 uses `npm` to update its version (true, it uses a command named `install` to do that, but...) so that it won't complain when it is run.

"Tweak" was meant to be general enough to encompass "update and configure" without having to go into the details.  I felt like there should be a "divider" setting the `npm` "stuff" off from the `dnf` "stuff", and, I figured it was worth putting some text into it to indicate the purpose of the division.  I suppose I could have said `npm Stuff` instead of "Tweak npm", but that doesn't really seem any more satisfying.

Got any suggestions?

Reply:
"Configure" npm?